active-object/bin/src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+
```

Comment:
If there is no properties configuration required, can we discard this file and not include here?